### PR TITLE
1344/prominent filter button

### PR DESCRIPTION
**Pull request by ColinBuyck**
# Pull Request Template

## Issue Overview

This PR addresses #1344 

- [ ] This change addresses the issue in full
- [x] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

This PR included updates to the Filter button styling on the Detroit Public listings page as well as the content (icon and text). Additionally, the filter button will now drop down a row and expand to full-width on mobile screens.

Update: This ticket implements the font awesome icon on the filtering results page with the recently added FA support. Also, @sheneral said that this PR could move forward without translations since translations are in progress but currently a bit delayed and can be implemented afterwards.

## How Can This Be Tested/Reviewed?

This can be tested on the Detroit public site by going to the listings page and noticing the new "Find Listings for You" button at both desktop and mobile dimensions. Then add some filters and observe the updated icon on the filter button on the results page.
## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Review by emilyjablonski — CHANGES_REQUESTED** [review]
Screen Shot 2022-07-13 at 11 14 56 AM
The hover state for the filter button has the icon as a white color, but on detroit dev right now looks like there is no hover state on the filter button so I imagine it should stay black. Additionally the string in the design is not capitalized, but it is in the issue, so I would check in with Jesse super quick about whether or not it should be capitalized.

**Comment by ColinBuyck** [comment]
@emilyjablonski Good catches! I confirmed the string casing with Jesse and fixed the hover issue. Let me know if there's anything else I can fix!

File: ui-components/src/locales/bn.json [hunk 1]
```diff
@@ -1,1263 +1,1264 @@
 {
```
Comment (emilyjablonski) [inline ui-components/src/locales/bn.json:1]: Do you know what caused these diffs in `bn.json` and `ar.json`? Is it maybe a prettier setting you have or a tab size setting?

**Review by emilyjablonski — APPROVED** [review]
Looks like the formatting changes in the translation files are expected as dev is for some reason inconsistent with prettier 👍 LGTM

**Comment by seanmalbert** [comment]
@ColinBuyck , can you resolve the merge conflicts. You're good to merge after that.

**Pull request merged**